requirements-dev.txt

```diff
@@ -1,11 +1,3 @@
-# Development requirements
 flake8
 nose-cov
-python-coveralls
```

Comment:
I remove python-coveralls because is only used by travis It is directly used now in the travis.yml
The other ones I removed because they already exists in requirements.txt/in